code/datums/flock/flock.dm

```diff
@@ -33,6 +33,7 @@ var/flock_signal_unleashed = FALSE
 	///list of strings that lets flock record achievements for structure unlocks
 	var/list/achievements = list()
 	var/mob/living/intangible/flock/flockmind/flockmind
+	var/relay_in_progress = FALSE
```

Comment:
it's a little pedantic, but this should probably be `has_relay` or something, since it doesn't get set to `FALSE` after the relay is constructed and is therefore no longer in progress.

Reply:
in_progress was just referring to full life cycle of relay, from teal print to finish/destruction

Reply:
Oh OK, that makes sense to me